db/schema.rb

```diff
@@ -182,6 +182,14 @@
     t.index ["code"], name: "index_countries_on_code", unique: true
   end
 
+  create_table "create_mail_delivery_failures", force: :cascade do |t|
```

Comment:
Looks like there are some older versions of the table here in this file which will need to be removed.